src/main/java/uk/nhs/digital/nhsconnect/nhais/service/SequenceService.java

```diff
@@ -0,0 +1,43 @@
+package uk.nhs.digital.nhsconnect.nhais.service;
+
+import io.micrometer.core.instrument.util.StringUtils;
```

Comment:
These aren't the StringUtils you're looking for! 🤖  Probably use the one from Spring.

Reply:
wow, that's not what I wanted :)